packages/firestore/package.json

```diff
@@ -48,8 +52,10 @@
     "protobufjs": "6.8.8",
     "rollup": "1.28.0",
     "rollup-plugin-copy-assets": "1.1.0",
+    "rollup-plugin-json": "4.0.0",
     "rollup-plugin-node-resolve": "5.2.0",
     "rollup-plugin-replace": "2.2.0",
+    "rollup-plugin-terser": "5.1.2",
```

Comment:
Unless this version is important you could probably move up to terser 5.1.3 and not have 2 separate versions in yarn.lock

Reply:
Updated to 5.1.3. Thanks!